### DISCUSSION TITLE
Overriding default plugin configurations in v2.5

### DISCUSSION THREAD

**Discussion by tesarlukas** [NvChad/NvChad#2745] [Q&A]
Hi, I understand that most of plugins NvChad uses can be overriden through chadrc.lua thanks to its merging with nvconfig.lua. However I don't see nvim-tree property inside of nvconfig.lua and I'm not sure how to override the configuration for it. I guess installing it as a separate plugin and extending its config function through using NvChad's configs would be an option too, but that seems kinda weird to me so I must be missing something.

I just want to modify the width of the file tree so it can expand much wider (preferably so that it shows the whole names of displayable directory contents). I've done this previously through just overwriting stuff in core directory of v2.0 structure, but that's not an option anymore.

**Comment by siduck** [comment]
https://nvchad.com/docs/config/plugins

you just put the table in your plugins table. lazy.nvim handles the overriding part
